### DISCUSSION TITLE
Does the Spray module of peleMP support combustion

### DISCUSSION THREAD

**Discussion by RachelMeng0096** [AMReX-Combustion/PeleC#726] [Q&A] [answered]
Hello everyone, in PeleMP's spray test cases, reactions are not enabled. I am curious to know whether it can support combustion. Could someone enlighten me?
![image](https://github.com/AMReX-Combustion/PeleC/assets/153271449/421b1e8a-36e7-4407-859b-3a21366e16ec)

**Comment by marchdf** [comment]
@ldowen  do you have an answer for the above?

**Comment by ldowen** [comment, marked as answer]
Yes, gas phase reactions are supported for spray cases.
